### PR TITLE
Vertically align images & padding for mobile

### PR DESCRIPTION
**Pull request by martijnjanssen**
Improve styling slightly by adding some extra spacing and a more uniform alignment of images on the page.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -38,3 +38,9 @@ nav {
 .page-selector > *:not([selected]) {
   display: none !important;
 }
+
+@media (max-width: 600px) {
```
Comment (TimvdLippe) [inline index.css:42]: This should actually be `var(--content-width)`, but variables are not allowed in media queries. Maybe just always add the margin?

File: photo-gallery.js [hunk 2]
```diff
@@ -44,8 +57,12 @@ class FlitciePhotoGallery extends LitElement {
             return html`
               <a href="${actualPath}" on-click=${event => this.goToUrl(event, actualPath)}>
                 <h3>${title}</h3>
-                <img src="https://flitcie.ch.tudelft.nl/var/thumbs${imagePath}">
-              </a>
+                <div>
+                  <div>
```
Comment (TimvdLippe) [inline photo-gallery.js:61]: Can we do this without adding these extra divs?

**Review by TimvdLippe — CHANGES_REQUESTED** [review]

File: photo-gallery.js (hunk 2 above)
Comment (martijnjanssen) [reply photo-gallery.js:61]: I don't think so, since the behavior of the image in the div changes when the div is present.

**Comment by TimvdLippe** [comment]
Manually merged in https://github.com/WISVCH/flitcie/commit/b05603b5d982d448ce023de8f26a390aeae2c0fb

**Pull request closed without merging**